documentation/asciidoc/topics/proc_configuring_sites_automatically.adoc

```diff
@@ -12,14 +12,24 @@ Configure {ispn_operator} to establish cross-site views with {brandname} cluster
 
 . Create an `Infinispan` CR for each {brandname} cluster.
 . Specify the name of the local site with `spec.service.sites.local.name`.
-. Set the value of the `service.sites.local.expose.type` field to either `NodePort` or `LoadBalancer`.
+. Set the value of the `spec.service.sites.local.expose.type` field to either `NodePort` or `LoadBalancer`.
+. Configure ports with the following fields as appropriate:
 +
-If you use a `NodePort` service for cross-site replication, you can configure a static port within the default range of `30000` to `32767` as follows:
-+
-[source,options="nowrap",subs=attributes+]
-----
-include::yaml/xsite_expose_nodeport.yaml[]
-----
+[%header,cols=3*]
+|===
+|Expose type
+|Field
+|Description
+
+|`NodePort`
+|`spec.service.sites.local.expose.nodePort`
+|Specifies a static port within the default range of `30000` to `32767`
```

Comment:
here is where the platform selects a port if not set of the if the port selected is not available (i.e. already in use by another service)
*I think*